test/e2e/wait_timeout_test.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	uitest "github.com/cppforlife/go-cli-ui/ui/test"
+)
+
+func TestWaitTimeout(t *testing.T) {
+	env := BuildEnv(t)
+	logger := Logger{}
+	kapp := Kapp{t, env.Namespace, env.KappBinaryPath, logger}
+
+	yaml1 := `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ns1
+  name: simple-app
+spec:
+  selector:
+    matchLabels:
+      simple-app: ""
+  template:
+    metadata:
+      labels:
+        simple-app: ""
+    spec:
+      containers:
+        - name: simple-app
+          image: docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+          env:
+            - name: HELLO_MSG
+              value: stranger
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns2
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ns3
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ns2
+  name: simple-app2
+spec:
+  selector:
+    matchLabels:
+      simple-app2: ""
+  template:
+    metadata:
+      labels:
+        simple-app2: ""
+    spec:
+      containers:
+        - name: simple-app2
+          image: docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+          env:
+            - name: HELLO_MSG
+              value: stranger
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ns3
+  name: simple-app3
+spec:
+  selector:
+    matchLabels:
+      simple-app3: ""
+  template:
+    metadata:
+      labels:
+        simple-app3: ""
+    spec:
+      containers:
+        - name: simple-app3
+          image: docker.io/dkalinin/k8s-simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+          env:
+            - name: HELLO_MSG
+              value: stranger
+`
+
+	name := "test-wait-timeout"
+	cleanUp := func() {
+		kapp.Run([]string{"delete", "-a", name})
+	}
+
+	cleanUp()
+	defer cleanUp()
+
+	logger.Section("Resource wait timeout", func() {
+		out, _ := kapp.RunWithOpts([]string{"deploy", "-f", "-", "-a", name, "--wait-timeout",
```

Comment:
should have _, err := ... and check that err is not nil.

Reply:
Done